packages/server/graphql/mutations/helpers/addSummariesToThreads.ts

```diff
@@ -20,10 +21,9 @@ const addSummariesToThreads = async (
   const commentPromises = stages.map(async (stage) => {
     const group = groups.find((group) => group.id === stage.reflectionGroupId)
     if (!group?.summary) return
-    const html =
-      tier === 'personal'
-        ? `<html><body><i>AI-generated summaries are a premium feature. We'll share these with you in your first few retros so you can see what they're like.</i><br><p><b>Topic Summary:</b></p><p>${group.summary}</p></body></html>`
-        : `<html><body><p><b>Topic Summary:</b></p><p>${group.summary}</p></body></html>`
+    const explainerText =
```

Comment:
+1 It would be nice to only show the explainer on the first topic as it gets quite repetitive after that.

Reply:
Makes sense! I've pushed an update so it only shows up on the first stage